src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -181,14 +181,13 @@ class PillSelector extends React.Component<Props, State> {
 
         return (
             <Tooltip isShown={!!error} text={error || ''} position="middle-right" theme="error">
+                {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
                 <span
                     className={classes}
                     onBlur={this.handleBlur}
                     onClick={this.handleClick}
                     onFocus={this.handleFocus}
                     onKeyDown={this.handleKeyDown}
-                    role="button"
-                    tabIndex={0}
```

Comment:
This was causing tabbing to require a second keystroke before the input got focus. I did some digging and it seems like the most likely reason that this was added was to avoid the eslint warning above.